aws/resource_aws_redshift_parameter_group.go

```diff
@@ -161,6 +166,19 @@ func resourceAwsRedshiftParameterGroupUpdate(d *schema.ResourceData, meta interf
 		d.SetPartial("parameter")
 	}
 
+	arn := arn.ARN{
```

Comment:
This seems like it would be super helpful to add as a resource attribute as well! Can you please use this logic to `d.Set("arn", arn)` in the `Read`, add the schema definition (with documentation), and then use `d.Get("arn").(string)` in the `setTagsRedshift()` call below? Thanks!

Reply:
Thanks, I fixed it and affected resources with this change.
acctest is working on now.